apiaudio/api_request.py

```diff
@@ -94,19 +94,37 @@ def _put_request_s3(cls, data, url=None, headers=None):
         # since aws s3 does not return a body on PUT requests,
         # r.json() does not work here
         return r
-    
+
     @classmethod
     def _put_request(cls, json, url=None, headers=None):
         headers = headers or {}
         url = url or f"{apiaudio.api_base}{cls.resource_path}"
         headers.update(cls._build_header())
-        
+
         r = requests.put(url=url, headers=headers, json=json)
 
         cls._expanded_raise_for_status(r)
 
-        if r.headers["Content-Type"] != "application/json":
+        if "application/json" not in r.headers["Content-Type"]:
+            return r.content
+        return r.json()
+
+    @classmethod
```

Comment:
why patch and not put?

Reply:
patch is more surgical, as we're changing particular fields here.

put is simply overwriting a whole entity through its URI